src/Synth/ADnote.cpp

```diff
@@ -1452,125 +1689,192 @@ inline void ADnote::ComputeVoiceOscillatorRingModulation(int nvoice)
 }
 
 /*
- * Computes the Oscillator (Phase Modulation or Frequency Modulation)
+ * Computes the Oscillator (WaveTable Modulation)
  */
-inline void ADnote::ComputeVoiceOscillatorFrequencyModulation(int nvoice,
-                                                              FMTYPE FMmode)
+inline void ADnote::ComputeVoiceOscillatorFrequencyOrWtModulation(int nvoice, FMTYPE FMmode)
```

Comment:
At least reading through this code it looks like only wavetable modulation uses the wavetable, so should this be split into one method for everything except that modulation type (which would look almost identical to the old code) and the new wavetable modulation version?